### PR TITLE
CAL-291 fixed formatting for sample Alliance urls in docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Aligns with changes to displaying default URLs in documentation. See also PR https://github.com/codice/ddf/pull/2955

#### Who is reviewing it? 
@mcalcote @vinamartin @ahoffer @garrettfreibott 

#### Choose 2 committers to review/merge the PR.
@clockard
@coyotesqrl
@lessarderic
@ricklarsen - Documentation
@rzwiefel
@shaundmorris

#### What are the relevant tickets?

[CAL-291](https://codice.atlassian.net/browse/CAL-291)
#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_transformers/chipping-metacard-transformer.adoc [hunk 1]
```diff
@@ -5,6 +5,8 @@
 :link: _chipping_metacard_transformer
 :summary: Extracts chips from NITF images.
 
+DeeDeeEff Voision ${ddf.version}
```
Comment (rzwiefel) [inline distribution/docs/src/main/resources/content/_transformers/chipping-metacard-transformer.adoc]: .

File: pom.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <ddf.version>2.11.5</ddf.version>
+        <ddf.version>2.11.6-SNAPSHOT</ddf.version>
```
Comment (rzwiefel) [inline pom.xml]: Ahhh did we forget to move the version forward after the release?

File: distribution/docs/src/main/resources/content/_transformers/chipping-metacard-transformer.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_transformers/chipping-metacard-transformer.adoc]: oops

File: pom.xml (hunk 2 above)
Comment (ricklarsen) [reply pom.xml]: `forgot` is maybe not the correct word

**Comment by ricklarsen** [comment]
build now

**Pull request merged**